http4s-backend/src/main/scala/sttp/client3/http4s/Http4sBackend.scala

```diff
@@ -36,12 +36,14 @@ class Http4sBackend[F[_]: Async](
   override def send[T, R >: PE](r: Request[T, R]): F[Response[T]] =
     adjustExceptions(r) {
       val (entity, extraHeaders) = bodyToHttp4s(r, r.body)
+      val version = r.httpVersion.map(versionToHttp4s).getOrElse(http4s.HttpVersion.`HTTP/1.1`)
```

Comment:
can we not set the version if its not explicitly set?

Reply:
It defaults to http4s.HttpVersion.`HTTP/1.1` anyway as far as i have checked

Reply:
well our contract is that we force the setting if present, otherwise we leave the default; if http4s at some point changes defaults, our code would become broken